tests/controllers/villains.test.js

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable max-len */
+const chai = require('chai')
+const sinon = require('sinon')
+const sinonChai = require('sinon-chai')
+const models = require('../../models')
+const {
+  afterEach, before, beforeEach, describe, it
+} = require('mocha')
+const { villainsList, singleVillain, postedVillain } = require('../mocks/villains')
+const { getAllVillains, getVillainBySlug, saveNewVillain } = require('../../controllers/villains')
+
+chai.use(sinonChai)
+const { expect } = chai
+
+describe('Controllers - villains', () => {
+  let sandbox
+  let stubbedFindOne
+  let stubbedFindAll
+  let stubbedSend
+  let response
+  let stubbedSendStatus
+  let stubbedStatusSend
+  let stubbedStatus
+  let stubbedCreate
+
+  before(() => {
+    sandbox = sinon.createSandbox()
+
+    stubbedFindOne = sandbox.stub(models.villains, 'findOne')
+    stubbedFindAll = sandbox.stub(models.villains, 'findAll')
+    stubbedCreate = sandbox.stub(models.villains, 'create')
+
+    stubbedSend = sandbox.stub()
+    stubbedSendStatus = sandbox.stub()
+    stubbedStatusSend = sandbox.stub()
+    stubbedStatus = sandbox.stub()
+
+    response = {
+      send: stubbedSend,
+      sendStatus: stubbedSendStatus,
+      status: stubbedStatus,
+    }
+  })
+
+  beforeEach(() => {
+    stubbedStatus.returns({ send: stubbedStatusSend })
+  })
+
+  afterEach(() => {
+    sandbox.reset()
+  })
+
+  describe('getAllVillains', () => {
+    it('retrieves a list of villains from the database and calls response.send() with the list', async () => {
+      stubbedFindAll.returns(villainsList)
+
+      await getAllVillains({}, response)
+
+      expect(stubbedFindAll).to.have.callCount(1)
```

Comment:
This is correct for your version of `findAll()` however that find should also have an `attributes` parameter to only return the expected data and in that case you would assert that it was called with that here.